test/evergreen/install_gcp_dependencies.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+install_abseil ()
+{
+    mkdir -p Downloads/abseil-cpp && cd Downloads/abseil-cpp
+    curl -sSL https://github.com/abseil/abseil-cpp/archive/20230125.0.tar.gz | \
+        tar -xzf - --strip-components=1 && \
+        sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h" && \
+        . ../../test/evergreen/find_cmake.sh
+        mkdir cmake-out && cd cmake-out
+            /opt/cmake/bin/cmake \
```

Comment:
To check. Do we need to call find_cmake.sh above if we're using the absolute path `/opt/cmake/bin/cmake` here?